src/main/resources/static/api.yml

```diff
@@ -243,6 +269,22 @@ components:
         - organisationId
         - contactEmail
 
+    OfferingRecord:
+      type: object
+      title: 'OfferingRecord'
+      properties:
+        course:
+          type: string
+        organisation:
+          type: string
+        "contact email":
+          type: string
+        prisonId:
+          type: string
+      required:
+        - course
+        - prisonId
```

Comment:
I think contact email should be required here - if we can't display a contact email at this stage of "Find", people can't refer people in prison onto the course. Are there examples where this is missing?

Reply:
Will adjust after merge to main.